exams/signals.py

```diff
@@ -29,10 +29,10 @@ def update_exam_authorization_final_grade(sender, instance, **kwargs):  # pylint
     authorize_for_exam(mmtrack, instance.course_run)
 
 
-@receiver(post_save, sender=CachedEnrollment, dispatch_uid="update_exam_authorization_cached_enrollment")
-def update_exam_authorization_cached_enrollment(sender, instance, **kwargs):  # pylint: disable=unused-argument
+@receiver(post_save, sender=CachedCurrentGrade, dispatch_uid="update_exam_authorization_grades")
```

Comment:
actually we determine if user is pass or not using a process that involves current grade 
https://github.com/mitodl/micromasters/blob/master/dashboard/utils.py#L223-L233

I think i should also add a signal for cached certificates as well, because for not financial_aid programs